keystorage/aws/src/test/java/tech/pegasys/signers/aws/SecretValueConverterUtilTest.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2023 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.signers.aws;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static tech.pegasys.signers.aws.SecretValueConverterUtil.mapSecretValue;
+
+import java.util.Set;
+
+import org.junit.jupiter.api.Test;
+
+class SecretValueConverterUtilTest {
+
+  @Test
+  void singleValueIsMapped() {
+    Set<String> mappedValues = mapSecretValue((k, v) -> v, "key", "value");
+    assertThat(mappedValues).containsOnly("value");
+  }
+
+  @Test
+  void newlinesValuesAreMapped() {
```

Comment:
Maybe a test case where a possibly "buggy" secret value is store with a starting \n? If that makes sense

Reply:
I'll add one ... it'll return an empty string, which will fails in our actual mapper in Web3Signer which uses BLS key mapping to return null.

Reply:
Along the same line of thinking, what if the file has lots of new lines?

Reply:
@siladu just pushed a test case to cover it ... basically all empty `\n` will be converted to empty strings .. which in case of web3signer will return null because BLS private key conversion will fail.